<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BeamProfilerDlg</class>
 <widget class="QDialog" name="BeamProfilerDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>900</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>光束质量分析</string>
  </property>
  <widget class="QFrame" name="frame_2">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>30</y>
     <width>651</width>
     <height>760</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <property name="lineWidth">
    <number>1</number>
   </property>
   <widget class="QPushButton" name="captureBtn">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>130</y>
      <width>160</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>手动采集图像</string>
    </property>
   </widget>
   <widget class="QGraphicsView" name="cameraGV">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>220</y>
      <width>500</width>
      <height>500</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="textLabel_8">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>20</y>
      <width>100</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Adobe Heiti Std</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>当前位置：</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="connBtn_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>70</y>
      <width>160</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>手动指定位移:</string>
    </property>
   </widget>
   <widget class="QLabel" name="textLabel_9">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>20</y>
      <width>100</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Adobe Heiti Std</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox_3">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>70</y>
      <width>60</width>
      <height>40</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
   </widget>
   <widget class="QLabel" name="textLabel_10">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>180</y>
      <width>160</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Adobe Heiti Std</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>当前图像展示：</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>461</width>
     <height>271</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="serialStatusLabel">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>160</y>
      <width>400</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Adobe Heiti Std</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(170, 0, 0);</string>
    </property>
    <property name="text">
     <string>位移平台：未连接</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="serialConnBtn">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>200</y>
      <width>140</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>连接平移台</string>
    </property>
   </widget>
   <widget class="QPushButton" name="serialFetchBtn">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>200</y>
      <width>120</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>刷新</string>
    </property>
   </widget>
   <widget class="QComboBox" name="serialportBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>200</y>
      <width>120</width>
      <height>40</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="cameraBtn">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>90</y>
      <width>140</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>连接相机</string>
    </property>
   </widget>
   <widget class="QLabel" name="cameraStatusLabel">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>50</y>
      <width>400</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Adobe Heiti Std</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(170, 0, 0);</string>
    </property>
    <property name="text">
     <string>相机状态：未连接</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="textLabel_11">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>100</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Adobe Heiti Std</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>初始化</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="frame_3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>330</y>
     <width>461</width>
     <height>541</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QPushButton" name="workdirBtn">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>180</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Adobe Heiti Std</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>选择工作目录</string>
    </property>
   </widget>
   <widget class="QLabel" name="textLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>140</y>
      <width>160</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Adobe Heiti Std</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>已采集：</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="textLabel_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>190</y>
      <width>160</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Adobe Heiti Std</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>采样帧率：</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="textLabel_7">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>290</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Adobe Heiti Std</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>ms</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="shitBtn">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>240</y>
      <width>120</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>暂不支持</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox_2">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>290</y>
      <width>60</width>
      <height>40</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
   </widget>
   <widget class="QLabel" name="textLabel_6">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>290</y>
      <width>160</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Adobe Heiti Std</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>曝光时间：</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="textLabel_5">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>240</y>
      <width>160</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Adobe Heiti Std</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>位移路径：</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="textLabel_4">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>190</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Adobe Heiti Std</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Hz</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="textLabel_2">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>140</y>
      <width>100</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Adobe Heiti Std</family>
      <pointsize>12</pointsize>
      <underline>true</underline>
     </font>
    </property>
    <property name="text">
     <string>0组数据</string>
    </property>
   </widget>
   <widget class="QPushButton" name="processDirBtn_3">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>450</y>
      <width>371</width>
      <height>60</height>
     </rect>
    </property>
    <property name="text">
     <string>计算结果</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>190</y>
      <width>60</width>
      <height>40</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
   </widget>
   <widget class="QPushButton" name="processDirBtn_2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>360</y>
      <width>160</width>
      <height>60</height>
     </rect>
    </property>
    <property name="text">
     <string>启动</string>
    </property>
   </widget>
   <widget class="QLabel" name="snapPathLabel">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>80</y>
      <width>260</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Adobe Heiti Std</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>工作目录未指定</string>
    </property>
   </widget>
   <widget class="QPushButton" name="processDirBtn">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>360</y>
      <width>160</width>
      <height>60</height>
     </rect>
    </property>
    <property name="text">
     <string>重置</string>
    </property>
   </widget>
   <widget class="QLabel" name="textLabel_12">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>100</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Adobe Heiti Std</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>主控</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
